Topic Wise Questions with Solutions/Array/Rearrange_the_neagtive_positive_element.py

```diff
@@ -0,0 +1,54 @@
+"""
+Given an array of positive and negative numbers, arrange them in an alternate 
+fashion such that every positive number is followed by negative and vice-versa 
+maintaining the order of appearance. 
+Number of positive and negative numbers need not be equal. 
+If there are more positive numbers they appear at the end of the array.
+If there are more negative numbers, they too appear in the end of the array.
```

Comment:
It says if the negative numbers is more then they appear in the end , that means positive numbers appear infront

Reply:
yes, they have appeared on the front